lamindb/dev/_run_context.py

```diff
@@ -32,15 +32,6 @@
 )
 
 
-def save_or_load(msg):
-    if logger._verbosity > 2:
-        logger.success(msg)
-    else:
-        logger.important(
-            msg.replace("loaded: ", "").replace("saved: ", "").replace("updated: ", "")
```

Comment:
@sunnyosun, removing this as, over the past months of this having been live, I felt it's really irritating to not know whether a transform or run are loaded or freshly created